### DISCUSSION TITLE
what is 2x output by nearest neighbor??

### DISCUSSION THREAD

**Discussion by marine025** [JargeZ/vhs#88] [General]
update august 2023: now i know what is it, its if you wanna render real 480 or only fictional 480, and the same with other qualities, its a very advanced ption because happened the same with real vhs's sometines looks great but at the time of digitalizing it looks so downgraded that even effects arent well apprecitaed, dissabling x2 will do the vidoe render longer and it will less size but it will have less quality just literally the quality you set 360 480 600 720..

is it a video rendering faster option or it means the video feels more speed than normal, more speed than the original video?

**Comment by SergeyMC9730** [comment]
it just upscales the image i think
